Assets/Scripts/Terrain/TerrainHelper.cs

```diff
@@ -404,52 +431,27 @@ public static void LayoutNatureBillboards(DaggerfallTerrain dfTerrain, Daggerfal
                     // Rect is expanded slightly to give extra clearance around locations
                     tilePos.x = x;
                     tilePos.y = y;
-                    const int natureClearance = 4;
-                    Rect rect = dfTerrain.MapData.locationRect;
-                    if (rect.x > 0 && rect.y > 0)
-                    {
-                        rect.xMin -= natureClearance;
-                        rect.xMax += natureClearance;
-                        rect.yMin -= natureClearance;
-                        rect.yMax += natureClearance;
-                        if (rect.Contains(tilePos))
-                            continue;
-                    }
-
-                    // Chance scaled based on map pixel height
-                    // This tends to produce sparser lowlands and denser highlands
-                    // Adjust or remove clamp range to influence nature generation
-                    float elevationScale = (dfTerrain.MapData.worldHeight / 128f);
-                    elevationScale = Mathf.Clamp(elevationScale, 0.4f, 1.0f);
-
-                    // Chance scaled by base climate type
-                    float climateScale = 1.0f;
-                    DFLocation.ClimateSettings climate = MapsFile.GetWorldClimateSettings(dfTerrain.MapData.worldClimate);
-                    switch (climate.ClimateType)
-                    {
-                        case DFLocation.ClimateBaseType.Desert:         // Just lower desert for now
-                            climateScale = 0.25f;
-                            break;
-                    }
+                    if (rect.Contains(tilePos))
```

Comment:
I need to add the 'null' rect test back in here otherwise it's calling contains for non-location tiles.